projects/ngx-meta/src/core/src/utils/make-injection-token.ts

```diff
@@ -27,9 +27,13 @@ export const _makeInjectionToken: <T>(
   const injectionToken =
     INJECTION_TOKENS.get(description) ??
     new InjectionToken(`ngx-meta ${description}`, { factory })
+  INJECTION_TOKENS.set(description, injectionToken)
```

Comment:
Moved here so the prod code is ran as soon as possible (after thinking about race conditions and so). Though not needed actually.